src/main/java/team/baymax/model/appointment/AppointmentStatus.java

```diff
@@ -1,5 +1,21 @@
 package team.baymax.model.appointment;
 
 public enum AppointmentStatus {
-    DONE, MISSED, UPCOMING;
+    DONE("done", "#3e9149"), MISSED("missed", "#ac4848"), UPCOMING("upcoming", "#4d66bc");
```

Comment:
I think we shouldn't put colour here, as colour is a UI atttribute. Instead we can create an AppointmentStatusColour that maps AppointmentStatus to the colour, so that we separate the UI from the model.